Az.DevTestLabs2.psm1

```diff
@@ -494,6 +512,10 @@ function New-AzDtlLab {
     [ValidateNotNullOrEmpty()]
     [string] $ResourceGroupName,
 
+    [parameter(Mandatory=$false,HelpMessage="IP address prefix (CIDR notation) within 10.0.0.0/20 reserved for VMs creation. (e.g 10.0.0.0/21, 10.)")]
+    [ValidateNotNullOrEmpty()]
+    [string] $VmCreationSubnetPrefix,
```

Comment:
By default a DevTest Lab is created in a subnet range 10.0.0.0/20 filling all the available VNet address space. This is the only viable solution I could find for reserving some unallocated address range for next bastion subnet creation. No DTL arm template supports specifying an address range or existing VNet during lab creation.